docs-source/content/userguide/managing-domains/domain-in-image/base-images/_index.md

```diff
@@ -88,38 +88,51 @@ Oracle Container Registry.
 
 #### Creating a custom image with patches applied
 
-The Oracle WebLogic Server Kubernetes Operator requires patch 29135930.
-This patch does have some prerequisite patches that will also need to be applied. The standard image `container-registry.oracle.com/middleware/weblogic:12.2.1.3` already has all of these patches applied.
+The Oracle WebLogic Server Kubernetes Operator and WebLogic Server 12.2.1.3.0 image requires patch 29135930.
+This patch has some prerequisite patches that will also need to be applied. To create and customize the WebLogic Server image,
+and apply the required patches, use the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool).
 
-[This sample](https://github.com/oracle/docker-images/blob/master/OracleWebLogic/samples/12213-patch-wls-for-k8s/README.md) in
-the Oracle GitHub Docker images repository demonstrates how to create an image with arbitrary patches, starting from an unpatched WebLogic Server 12.2.1.3 image (not the standard `container-registry/middleware/weblogic:12.2.1.3` pre-patched image).  You can customize that sample to apply a different set of patches, if you require additional patches or PSUs.
+To use the Image Tool, follow the [Setup](https://github.com/oracle/weblogic-image-tool/blob/master/README.md#setup) instructions
+and [Quick Start](https://github.com/oracle/weblogic-image-tool/blob/master/site/quickstart.md) Guide.
 
-When using that sample, you will need to download the required patch and also
-some prerequisite patches.  To find the correct version of the patch, you should
-use the "Product or Family (Advanced)" option, then choose "Oracle WebLogic Server"
-as the product, and set the release to "Oracle WebLogic Server 12.2.1.3.181016" as
-shown in the image below:
+To build the WebLogic Server image and apply the patches:
 
-![patch download page](/weblogic-kubernetes-operator/images/patch-download.png)
+1. Add the Server JRE and the WebLogic Server installer to the [`cache` command](https://github.com/oracle/weblogic-image-tool/blob/master/site/cache.md).
 
+    ```
+    $ imagetool cache addInstaller \
+    --type jdk \
+    --version 8u241 \
+    --path /home/acmeuser/wls-installers/jdk-8u202-linux-x64.tar.gz
```

Comment:
the server JRE file name usually starts with "server" and not "jdk", like "server-jre-8u202-linux-x64.tar.gz" . Shouldn't we use that as the example since server JRE is what you are recommending?

Reply:
Server JRE is used in WLS 12.2.1.x but in 14 it will be JDK

Reply:
sure lets use JRE for now